### ISSUE TITLE
Unable to use installation script

### ISSUE THREAD

**Issue by candrewlee14** [candrewlee14/webman#46]
I'm running into an error installing a amd64 linux machine.
Output:
```
curl https://raw.githubusercontent.com/candrewlee14/webman/main/scripts/install.sh | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  9690  100  9690    0     0   126k      0 --:--:-- --:--:-- --:--:--  126k
candrewlee14/webman info checking GitHub for latest tag
candrewlee14/webman info found version: 0.9.0 for v0.9.0/linux/x86_64
Refreshing package recipes for "webman"...
rename /home/candrewlee14/.webman/tmp/recipes/webman-pkgs-main /home/candrewlee14/.webman/recipes/webman: no such file or directory
webman: open /home/candrewlee14/.webman/recipes/webman/pkgs/webman.webman-pkg.yml: no such file or directory
Not all packages installed successfully
```

@jolheiser I think this has to do with the multi-repo configuration.

**Comment by jolheiser** [comment]
Yep, this should be fixed when a new release is cut, this was fixed in #43

**Comment by candrewlee14** [comment]
Oh gotcha, thanks for clearing that up. I didn't realize those were the same problem :)
